src/tests/ftest/container/fill_destroy_loop.py

```diff
@@ -96,7 +95,9 @@ def write_pool_until_nospace(self, test_loop):
 
             try:
                 container.write_objects()
-            except TestFail as excep:
+            except DaosTestError as excep:
+                # Uncomment following for debugging
```

Comment:
Do you want to uncomment these lines?

Reply:
I hesitated to remove these debugging lines.
However, they were introduced by @daltonbohning and I am not sure if it is on purpose or an oversight.
@daltonbohning , could you tell me if we should keep it ?

Reply:
I didn't add it, but I do think it can be removed

Reply:
My bad @daltonbohning , I missinterpreted the git blame when I resolved the last merging issue.
This code was in place from the begining: https://github.com/daos-stack/daos/commit/57bbb03f1a8a69b103b372797f1e483b421df459.
I will remove it and repush: functional tests have not yet started.